central/imageintegration/store/defaults.go

```diff
@@ -171,5 +173,9 @@ var (
 			_, creator := clairify.Creator(nil)
 			return creator
 		}),
+		makeDelayedIntegration(DefaultScannerV4Integration, func() scanners.Creator {
+			_, creator := scannerv4.Creator(nil)
+			return creator
+		}),
```

Comment:
Why was Scanner V4 added as a delayed integration? This was purposely avoided during implementation.  The Scanner V4 integration is created by default (based on feature being enabled) - was there an issue with the prior mechanism?

The Scanner V2 delayed integration has caused more issues than it solves in practice - this was avoided for Scanner V4 on purpose.

Reply:
Reverted in https://github.com/stackrox/stackrox/pull/13040